packages/plugins/GoPlusSecurity/src/SNSAdaptor/components/SecurityPanel.tsx

```diff
@@ -209,7 +210,7 @@ export const SecurityPanel = memo<TokenCardProps>(({ tokenSecurity, tokenInfo, t
                 </Stack>
                 <Stack className={classes.detectionCollection} sx={{ maxHeight: LIST_HEIGHT.max, overflowY: 'auto' }}>
                     {makeMessageList.map((x, i) => (
-                        <RiskCard tokenSecurity={tokenSecurity} info={x} key={i} />
+                        <RiskCard tokenSecurity={tokenSecurity} info={x as unknown as SecurityMessage} key={i} />
```

Comment:
Why this change?

Reply:
`RiskCard` is a component provided by @masknet/shared/UI, which has a different i18N field. This problem will be solved after Token Security Refactor